doc/api.md

```diff
@@ -237,6 +241,12 @@ Note that, when information coming from devices, this means measures, are not de
 device, the IoT agent will store that information into the destination entity using the same attribute name than the
 measure name, unless `explicitAttrs` is defined. Measures `id` or `type` names are invalid, and will be ignored.
 
+## Device autoprovision and entity creation
+
+For those agents that uses IoTA Node LIB version 3.4.0 or higher, you should consider that the entity is not created
+automatically when a device is created. This means that all entities into the Context Broker are created when data 
+arrives from a device, no matter if the device is explicitly provisioned (via [device provisioning API](#create-device-post-iotdevices)) or autoprovisioned.
+
```

Comment:
Maybe we should a paragraph explaining what to do if you need the entity in advance? Something like this:

> If for any reason you need the entity at CB before the first measure of the corresponding device arrives to the IOTAgent, you can create it in advance using the CB [NGSIv2 API](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md).

Please, adapt/improve the text as needed.

Reply:
Added in 557f75aa143a9430a92ef44859a61bb8f6c22797